js/script.js

```diff
@@ -0,0 +1,36 @@
+const taskInput = document.getElementById("task-input");
+const addButton = document.getElementById("btn-add");
+const unoderedList = document.getElementById("tasks");
+const notification = document.getElementById("message");
+
+let tasks = [];
+const addTaskHandler = () => {
```

Comment:
```suggestion
let tasks = [];

const addTaskHandler = () => {
```

Reply:
Fixed the issue.

Reply:
I suggested an empty line before the function. Just for better outlook of the codebase.